src/Service.GraphQLBuilder/Mutations/CreateMutationBuilder.cs

```diff
@@ -212,13 +426,13 @@ private static ITypeNode GenerateListType(ITypeNode type, ITypeNode fieldType)
         /// </summary>
         /// <param name="typeName">Name of the entity</param>
         /// <returns>InputTypeName</returns>
-        private static NameNode GenerateInputTypeName(string typeName)
+        public static NameNode GenerateInputTypeName(string typeName)
         {
             return new($"{EntityActionOperation.Create}{typeName}Input");
         }
 
         /// <summary>
-        /// Generate the `create` mutation field for the GraphQL mutations for a given Object Definition
+        /// Generate the `create` point/batch mutation fields for the GraphQL mutations for a given Object Definition
```

Comment:
Will this mutation field now allow bulk inserts then?

Reply:
Bulk won't be the terminology we will be using as that is a different concept with MsSql. It'll be called multiple create.